crates/nu-cmd-lang/src/core_commands/commandline.rs

```diff
@@ -64,7 +69,32 @@ impl Command for Commandline {
                 .lock()
                 .expect("repl cursor pos mutex");
 
-            if call.has_flag("append") {
+            if call.has_flag("cursor") {
+                let arg_str = cmd.as_string()?;
+                match arg_str.parse::<i64>() {
+                    Ok(n) => {
+                        *cursor_pos = if n <= 0 {
+                            0usize
+                        } else {
+                            buffer
+                                .char_indices()
```

Comment:
This is a question. Wouldn't we want to be using graphemes here instead of char_indicies()?  See this https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0173f1b8e9f2c29d867a44122bb680f8

Reply:
Thanks, great idea.